README.md

```diff
@@ -46,7 +46,7 @@ Please find upgrade information to major versions here:
 ### :wrench: Tools
 
 - [CLI](https://github.com/sequelize/cli)
-- [With TypeScript](https://sequelize.org/docs/v7/other-topics/typescript)
+- [With TypeScript](https://sequelize.org/docs/v6/other-topics/typescript)
 - [Enhanced TypeScript with decorators](https://github.com/RobinBuschmann/sequelize-typescript)
```

Comment:
You can go ahead and delete these two lines. We've updated the docs and now TypeScript is shown in most of the examples and decorators are also included.

Reply:
Should I also remove line 50?

`- [Enhanced TypeScript with decorators](https://github.com/RobinBuschmann/sequelize-typescript)`